src/analyses/ai.cpp

```diff
@@ -187,12 +187,14 @@ void ai_baset::entry_state(const goto_programt &goto_program)
   get_state(goto_program.instructions.begin()).make_entry();
 }
 
-void ai_baset::initialize(const goto_functionst::goto_functiont &goto_function)
+void ai_baset::initialize(
+  const irep_idt &function,
```

Comment:
While doing this could we agree on a consistent name, e.g. `function_id`, `function_identifier`, `function_name` or so?

Reply:
Done (now using `function_identifier`).